r-package/grf/R/causal_forest.R

```diff
@@ -157,13 +157,13 @@ causal_forest <- function(X, Y, W,
 
     if (is.null(Y.hat) && !orthog.boosting) {
       forest.Y <- regression_forest(X, Y, sample.weights = sample.weights, sample.fraction = sample.fraction, mtry = mtry, tune.parameters = tune.parameters,
-                                    num.trees = min(500, num.trees), num.threads = num.threads, min.node.size = NULL, honesty = TRUE,
```

Comment:
It would be nice to pull this out into a variable, to avoid repeating the transformation in several different places.